### PR TITLE
New/order-transactions

### PR DESCRIPTION
**Pull request by matthew1809**
Adds ability to Get transactions associated with an order. Tested locally with success.

Tried to follow existing structures with exception of adding a nestedPropertyVal assertion due to there are multiple properties with the same name in the transaction object.

### CODE REVIEW

File: src/endpoints/orders.js [hunk 1]
```diff
@@ -14,6 +14,11 @@ class OrdersEndpoint extends BaseExtend {
   Payment(id, body) {
     return this.request.send(`${this.endpoint}/${id}/payments`, 'POST', body);
   }
+  
+  Transactions(id) {
+  return this.request.send(`${this.endpoint}/${id}/transactions`, 'GET');
```
Comment (outrunthewolf) [inline src/endpoints/orders.js]: formatting?

**Comment by ynnoj** [comment]
https://github.com/moltin/js-sdk#contributing

**Comment by ynnoj** [comment]
> Note: Commits should adhere to the Angular commit conventions.

😉 https://github.com/moltin/js-sdk#contributing

**Comment by ynnoj** [comment]
Also, maybe we should set up [prettier](https://prettier.io) to enforce listing standards 🤔

File: src/endpoints/orders.js (hunk 1 above)
Comment (matthew1809) [reply src/endpoints/orders.js]: https://github.com/moltin/js-sdk/pull/109/commits/7ed5090cf62461a83d02aff58e318c9d02c4b9a9

**Comment by ynnoj** [comment]
Waiting for #110 to land. Will refactor the additional test in this PR once merged.

**Comment by ynnoj** [comment]
👉 #111

**Pull request closed without merging**